File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        boolean unique = true;
        int position = 0;
        for(int i = 1; i<nums.length; i++){
            if(nums[i]>nums[i-1]){
                unique = true;
            }
            if(nums[i]==nums[i-1]){
                unique = false;
            }
            if(unique){
                nums[position+1]=nums[i];
                position++;
            }
        }
        return position+1;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        j = 0

        for i in range(1,len(nums)):
            if nums[i] > nums[i-1]:
                nums[j+1] = nums[i];
                j = j+1;
        return j+1        